src/components/content-box/_content-box.scss

```diff
@@ -17,6 +17,10 @@ $includeHtml: false !default;
         margin-top: - rhythm(0.5);
         margin-bottom: rhythm(0.5);
       }
+
+      &--large ~ .mint-content-box__actions--large {
```

Comment:
Not sure whether it can be a simple modifier. It depends on context.
But I like this idea more.